.kube/app/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM php:8.4-fpm
 
 ENV PHP_VERSION 8.4
+# TODO - IMAGICK_VERSION can be removed once pecl fixes issues installing with php8.4
+ENV IMAGICK_VERSION 3.7.0 # This should be bumped to the latest version, especially when PHP version is bumped and can be removed when https://github.com/accessibility-exchange/platform/issues/2425 is resolved
```

Comment:
I guess this link is a bit odd because this PR will close that issue. Maybe reword a bit here so that it indicates that 2425 is what this work around is for an can be removed when the upstream issue in imagick is resolved (and link to an imagick issue).

Reply:
Changed all the TODO comments to reference issue https://github.com/Imagick/imagick/issues/698.